src/io/flutter/sdk/FlutterSdkVersion.java

```diff
@@ -89,6 +94,11 @@ public boolean isTrackWidgetCreationRecommended() {
     return version != null && version.compareTo(MIN_SAFE_TRACK_WIDGET_CREATION_SDK.version) >= 0;
   }
 
+  public boolean isAndroidxSupported() {
+    assert (MIN_ANDROIDX_SDK.version != null);
```

Comment:
nit: remove assert that MIN_ANDRIOD_SDK.version is not null.

Reply:
Disabled the inspection for this statement.